addons/account/models/account_move.py

```diff
@@ -4394,10 +4395,60 @@ def _set_tax_cash_basis_entry_date(self, move_date, newly_created_move):
             # probably already sent to the estate.
             newly_created_move.write({'date': move_date})
 
+    def _get_tax_cash_basis_base_key(self, tax, move, line):
+        account_id = self._get_tax_cash_basis_base_account(line, tax)
+        tax_rep_lines = tax.refund_repartition_line_ids if line.move_id.type in ('in_refund', 'out_refund') else tax.invoice_repartition_line_ids
+        original_base_tags = tax_rep_lines.filtered(lambda x: x.repartition_type == 'base').tag_ids
+        base_tags = tuple(line._convert_tags_for_cash_basis(original_base_tags).ids)
+        return (line.id, account_id.id, tax.id, line.tax_repartition_line_id.id, base_tags,line.currency_id.id, line.partner_id.id, line.move_id.type)
```

Comment:
Why is the key different from 12.3? In 12.3:
https://github.com/odoo/odoo/blob/3dc75d63a3f41977d0e081ae3252fc5f31280f08/addons/account/models/account_move.py#L1849

Reply:
because in the new Journal Entry the tax_repartition is actually taken into account in v13.0.

That is the reason.

Regards.

Reply:
https://github.com/odoo/odoo/blob/saas-12.3/addons/account/models/account_move.py#L1863-L1871

https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L4479

Reply:
@nim-odoo any news?

Regards.